### ISSUE TITLE
-k1 not work on windows build

### ISSUE THREAD

**Issue by DeniR** [powturbo/TurboBench#47]
Option -k# (default repeat count 3 ) not work on windows build, "turbobench.exe testme.tar -k1" show help page instead bench.

**Comment by powturbo** [comment]
Yes, it's removed. You can use repeats with -I# for compression and -J# for decompression. 
Ex. "turbobench.exe testme.tar -I15 -J15
